modules/updateChecker.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+const _ = global._;
+const got = require('got');
+const semver = require('semver');
+const Windows = require('./windows');
+const log = require('./utils/logger').create('updateChecker');
+
+
+
+/**
+ * Check for updates to the app.
+ * @return {[type]} [description]
+ */
+const check = exports.check = function() {
+    log.info('Check for update...');
+
+    let str = null;
+
+    switch (global.mode) {
+        // right now we are just releasing the wallet!
+        case 'mist':
+        case 'wallet':
+            str = 'wallet';
+            break;
+    }
+
+    return got('https://api.github.com/repos/ethereum/mist/releases', {
+        timeout: 3000,
+        json: true,
+    })
+    .then((res) => {
```

Comment:
That was the intention (for the future) but I saw we haven't done a Mist version so merged them for now - maybe I'm wrong about that, is there a Mist version?